packages/cli/src/commands/cluster/add/command.ts

```diff
@@ -165,15 +166,36 @@ export class ClusterAddCommand extends PanfactumCommand {
       ]);
     }
 
+    /***********************************************
+     * Confirms the vCPU quota is high enough
+     ***********************************************/
+    const serviceQuotasClient = new ServiceQuotasClient({ region, profile: awsProfile })
+    const command = new GetServiceQuotaCommand({
+      QuotaCode: "L-1216C47A",
+      ServiceCode: "ec2",
+    })
+    try {
+      const quota = await serviceQuotasClient.send(command)
+      if (quota.Quota?.Value && quota.Quota.Value < 32) {
```

Comment:
We are good as long as they have at least 16 vcpu headroom. Can we do `limit - usage >= 16`?